crates/router/src/utils/currency.rs

```diff
@@ -282,57 +268,30 @@ async fn fallback_forex_redis_check(
         Some(redis_forex) => {
```

Comment:
Nit: The `is_redis_expired()` function (just before the `match` arm) made me think that it returned a `bool`.